scenario.py

```diff
@@ -407,12 +419,13 @@ def split_data(self, is_logging_enabled=True):
         # ... or receive different amounts?
 
         # Check the percentages of samples per partner and control its coherence
-        assert len(self.amounts_per_partner) == self.partners_count
-        assert np.sum(self.amounts_per_partner) == 1
+        assert len(self.amounts_per_partner) == self.partners_count, "Error: in .yml in amounts_per_partner arguments: the amounts_per_partners arguments (a list) has to contain a proportion per each partner - Here there is too many or not enough proportions"
```

Comment:
Would replace by `"amounts_per_partner list should have a size equals to partners_count"`